src/components/app.jsx

```diff
@@ -1,9 +1,163 @@
 import React from "react";
-import { ReactComponent as Logo } from "../images/sun.svg";
 import "../App.css";
+import { getWeather, getForecast } from '../utils.js';
+import Header from './Header';
+import WeatherCurrent from './WeatherCurrent';
+import WeatherForecast from './WeatherForecast';
 
-function App() {
-  return <h1>Weather</h1>;
+class App extends React.Component {
```

Comment:
Have you worked with React hooks before? If so, how do you decide to make class-based components vs functional components with hooks?

Reply:
I haven't worked with React hooks before, but it's one of the features I'm most interested in learning! I actually thought about trying it for the first time on this project, but then decided that maybe an interview project wasn't the best place to experiment. From what I've read, though, it seems like functional components with hooks should be the default over class components, as they require fewer lines of code to accomplish the same thing.